tools/webdriver/webdriver/client.py

```diff
@@ -326,9 +326,14 @@ def size(self):
     @command
     def size(self, new_size):
         """Set window size by passing a tuple of `(width, height)`."""
-        width, height = new_size
-        body = {"width": width, "height": height}
-        self.session.send_session_command("POST", "window/rect", body)
+        try:
+            width, height = new_size
+            body = {"width": width, "height": height}
+            self.session.send_session_command("POST", "window/rect", body)
+        except (error.UnknownErrorException, error.InvalidArgumentException):
+            # silently ignore this error as the command is not implemented
+            # for Android. Revert this once it is implemented.
+            pass
```

Comment:
Since it is not implemented in Android only, should it only make it pass on Android after catching the exception?

Reply:
I don't see how to do this. Also I believe we don't get exception here, as we did not capture it before, which implies this change is fine for other platforms also.